### ISSUE TITLE
Disconnect `Distribute Award` function from Group Inventory

### ISSUE THREAD

**Issue by M0nk3yy** [foundryvtt/dnd5e#4400]
If distributing an arbitrary reward to players such as 100 gp, the current workflow is:

1. Open `Group` sheet
2. Click `Inventory` tab
3. Add 400 gp of currency
4. Click back to `Members` tab
5. Click `Award`
6. *If* this is the only currency in the inventory, click `Award`, otherwise further adjustment steps are required.

This is (I think) needlessly convoluted. I suggest not requiring currency be present in the party inventory prior to distribution, allowing much faster ad hoc currency rewards. This would use the following workflow:

1. Open `Group` sheet
2. Click `Award`
3. Add 400 gp to `Distribute Award` window
4. Click `Award`

Whilst this method requires the awarded currency to be entered at playtime rather than in advance, this detriment is offset by the fact that multiple rewards can be awarded more smoothly.

This method particularly supports the use case where the Group Inventory is being used as ... well a group inventory. If the players are already storing some of their currency in the group actor, and the GM wishes to distribute some currency individually - they need to both add the awarded currency to the inventory and then modify all the currency values at time of award. Otherwise, they will be awarded their own original currency.

![image](https://github.com/user-attachments/assets/2e1292a5-c3fe-450d-ae8c-6814a21f3394)

Related conversation in #2923 with particular note to https://github.com/foundryvtt/dnd5e/issues/2923#issuecomment-1948633916 where @schultzcole suggests a "Distribute From Group" checkbox in the Distribute Award window. Whilst utimately I think disconnecting the group inventory from the award distribution is the better UX, this provides a good potential compromise if it's decided that the current implentation won't change.

**Comment by Fyorl** [comment]
While there is certainly more room for improvement in UX and discoverability here, I don't think this issue is framed at all accurately.

Awarding an arbitrary award of 100gp to each player would be accomplished via the following:

1. Type `/award 100gp each` in chat.
2. Click `Award`.

This is already a lot faster than the claimed 'current workflow', as well as the proposed improved workflow.

**Comment by M0nk3yy** [comment]
Yeah, I didn't feel that was worth mentioning for exactly the reasons you highlighted. `/award` is so undiscoverable it might as well not exist for the majority of players. I only found out about it when reading your github comment here https://github.com/foundryvtt/dnd5e/issues/2923#issuecomment-1948572351, so less clued-in users don't stand a chance.

This issue is really only targeting improvements to the Distribute Award method using a GUI, which I think it's fair to say would be the go-to option for the majority of users.

**Comment by Fyorl** [comment]
I think, when exploring solutions for distributing awards, we should consider all of the moving parts, rather than simply focusing on one perceived friction and attempting to resolve only that.

The `/award` command is the intended solution for ad-hoc awards. The group actor award button is intended to be used to distribute awards from encounter groups, or to distribute accumulated party awards.

We can add some option to make the group actor's award feature ad-hoc. We can also make the `/award` command more discoverable in some way. The `/award` command does spawn the Distribute Award dialog, so it is already partly a GUI solution, but if adding a button somewhere is necessary for it to be considered 'using a GUI', then that is also an option that would not involve having to open a group sheet at all.

**Comment by unsoluble** [comment]
While it’s not a _perfect_ fit for any ad-hoc reward, having an award button in the Combat Encounters tab would be pretty decent.

When an encounter exists, it would distribute the included monster XP (perhaps even accounting for Defeated automatically). When no encounter, it could act as an ad-hoc distribution.

**Comment by thatlonelybugbear** [comment]
Makes sense if that becomes associated with Combat Encounters, if it was to be implemented 🤔
